src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawingControllerAdapter.java

```diff
@@ -5,13 +5,17 @@
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+import javax.swing.*;
+
+import static edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController;
+
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawingControllerAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter.

Reply:
Inheritance is used once - in constructor.